docs/DeveloperGuide.md

```diff
@@ -16,6 +15,12 @@ optimised for the Command Line Interface (CLI), so that users can update and ret
 ### Purpose of Document
 This document specifies the architecture and software design for the application, tCheck.
 
+### Audience
+
+The Developer Guide is designed for those who are interested in understanding the architecture and other aspects of software design
+of tCheck. In particular, this guide has been written with the current and future tCheck developers in mind because it details
+the knowledge necessary to know to be able to modify the codebase and customize tCheck for specific operational needs or extend current functionalities.
```

Comment:
Incorrect grammar. "... knowledge necessary to (be able to) modify ... "
The words in brackets are optional, you may add it or not. If you do, remove the brackets.

"customise" for british spelling.

Reply:
Noted, will change